codecov_auth/commands/owner/tests/test_owner.py

```diff
@@ -55,10 +55,10 @@ def test_save_terms_agreement_delegate_to_interactor(self, interactor_mock):
         self.command.save_terms_agreement(input_dict)
         interactor_mock.assert_called_once_with(input_dict)
 
-    @patch("codecov_auth.commands.owner.owner.StartTrialInteractor.execute")
-    def test_start_trial_delegate_to_interactor(self, interactor_mock):
+    @patch("codecov_auth.commands.owner.owner.CancelTrialInteractor.execute")
+    def test_cancel_trial_delegate_to_interactor(self, interactor_mock):
```

Comment:
hmm why did you replace it not add a new test here?

Reply:
cause I'm too fast for my own good and I messed up 😂 changing it